simpeg_drivers/params.py

```diff
@@ -356,6 +360,24 @@ class BaseInversionOptions(CoreOptions):
     percentile: float = 95.0
     epsilon_cooling_factor: float = 1.2
 
+    @property
+    def gradient_dip(self) -> np.ndarray | None:
+        """Gradient dip angle in clockwise radians from horizontal."""
+        if self.gradient_rotation is not None:
+            dip_uid = self.gradient_rotation.properties[1]
+            dips = self.geoh5.get_entity(dip_uid)[0].values
+            return np.deg2rad(dips)
+        return None
+
+    @property
+    def gradient_direction(self) -> np.ndarray | None:
+        """Gradient direction angle in clockwise radians from north"""
+        if self.gradient_rotation is not None:
+            direction_uid = self.gradient_rotation.properties[0]
+            directions = self.geoh5.get_entity(direction_uid)[0].values
+            return np.deg2rad(directions)
+        return None
```

Comment:
Recommend having a seperate method that will convert the input property group into a dip and azimuth angle, so we can generalize later with different group types.

Reply:
I added support for Strike & dip groups by simply subtracting 90 from the incoming direction.  Is there any reason to do  more than this?

Reply:
Will be for the conversion of Vector group to dip azimuth.

Reply:
To be addressed in GEOPY-2108